crates/ruff_python_ast_integration_tests/tests/visitor.rs

```diff
@@ -247,7 +247,7 @@ impl Visitor<'_> for RecordVisitor {
         self.exit_node();
     }
 
-    fn visit_except_handler(&mut self, except_handler: &ExceptHandler) {
+    fn visit_except_handler(&mut self, except_handler: &ExceptHandler, _is_star: bool) {
```

Comment:
I'd prefer if we can keep the `Visitor` trait unchanged. 

You can access the `try` statement from inside the rules using:

```
let is_star = checker
                    .semantic()
                    .current_statement()
                    .as_try_stmt()
                    .is_some_and(|try_stmt| try_stmt.is_star);
```

Reply:
Got it 👍 
I will be applying the changes.

Reply:
Thanks

Reply:
Change applied.
It's much simpler now!

I should spend sometime reading the common structs' definitions next time. :+1: